src/include/common/arrow/arrow_nullmask_tree.h

```diff
@@ -16,7 +16,7 @@ class ArrowNullMaskTree {
     bool isNull(int64_t idx) { return mask->isNull(idx + offset); }
     ArrowNullMaskTree* getChild(int idx) { return &(*children)[idx]; }
     ArrowNullMaskTree* getDictionary() { return dictionary.get(); }
-    ArrowNullMaskTree operator+(int64_t offset);
+    ArrowNullMaskTree offsetBy(int64_t offset);
```

Comment:
Why do we change the operator overloading to function?

Reply:
I felt that overloading the operator was vague and potentially misleading so I changed it to a named function